ec2/ec2-docker-setup-commands.sh

```diff
@@ -0,0 +1,6 @@
+docker pull ollama/ollama
+docker run -d --name ollama -p 11434:11434 ollama/ollama
+docker exec -it ollama ollama run llama3.2:1b
+curl http://localhost:11434/api/generate -d '{"model":"llama3.2:1b","prompt":"Why is the sky blue?", "stream": false}'
+
+docker start d2a556f0fab2
```

Comment:
wouldnt this container id be different for every build? are we using this script or just leaving this as reference